### PR TITLE
fix: grammar and a bit of wording

### PR DESCRIPTION
**Pull request by Sc4ramouche**
As I refer to the text of our hometask multiple times a week, I notice more and more grammar inconsistencies here and there. I thought I could fix them using an LLM, along with minor stylistic issues. 

Please let me know if you see these changes as beneficial. I personally think it does reduce the charmanté goofiness a bit, but overall this should help to make the task description slightly easier to follow. Generally I wanted to fix missing "a" and "the" articles, but other changes suggested by the LLM look ok too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,29 +10,29 @@
 
 ## Welcome to our little coding exercise! 👋
 
-Here you will be given the opportunity to play with TypeScript and Rockets in the same project. For this, we recommend you to block 2-3 hours of your time to complete it.
+Here, you will have the opportunity to work with TypeScript and Rockets in the same project. We recommend setting aside 2-3 hours to complete it.
 
-**Please make sure you have read this page entirely, before starting the challenge**. If you have any doubts, please reach out to us by [opening an issue](#how-to-request-help) as described at the bottom of this page.
+**Please make sure to read this page in its entirety before starting the challenge.** If you have any questions, feel free to [open an issue](#how-to-request-help) as described at the bottom of this page.
```
Comment (paul-pro) [inline README.md:15]:
```suggestion
**Please make sure to read this page entirely before starting the challenge.** If you have any questions, feel free to [open an issue](#how-to-request-help) as described at the bottom of this page.
```

File: README.md [hunk 2]
```diff
@@ -98,39 +98,39 @@ The expected result should be:
...
 - If no 'missions' are obtained from the custom hook, the component renders `"No data"` on the screen.
 
 **Expected outcome:**
 
-Please make sure to set the `filterParams` to match the example below:
+Please set the `filterParams` to match the example below:
 
 ```js
-<RocketList
+<RocketsList
```
Comment (paul-pro) [inline README.md:116]:
```suggestion
<RocketList
```
we aren't updating it, right?

File: README.md [hunk 3]
```diff
@@ -154,7 +154,7 @@ Describe your question(s) and [mention](https://docs.github.com/en/github/writin
 [spacex-api]: https://api.spacexdata.com/v3/launches/past
 [spacex-api-docs]: https://docs.spacexdata.com/?version=latest#fce450d6-e064-499a-b88d-34cc22991bcc
 [github-collaborators]: https://help.github.com/en/articles/inviting-collaborators-to-a-personal-repository
-[task-1]: #task-1---javascript
+[task-1]: #task-1---typescript
```
Comment (paul-pro) [inline README.md:157]: is it a correct update?

**Review by paul-pro — APPROVED** [review]
the rest lgtm

File: README.md (hunk 2 above)
Comment (Sc4ramouche) [reply README.md:116]: In the title of the task we ask to `Implement the `<RocketsList>` component`, so this change makes sense.

File: README.md (hunk 3 above)
Comment (Sc4ramouche) [reply README.md:157]: Yes, this updates the link to the task 1, which became `TypeScript` after this PR — https://github.com/adjust/fe-hometask/pull/7.

**Pull request merged**